app/pages/organizations/organization-page.jsx

```diff
@@ -110,6 +124,23 @@ class OrganizationPage extends React.Component {
               />
               <Translate content="organization.home.viewToggle" />
             </label>}
+          {this.props.organization.categories && this.props.organization.categories.length > 0 &&
+            <div className="organization-page__categories">
+              <button
+                className={this.calculateClasses('All')}
+                onClick={this.handleCategoryChange.bind(this, '')}
+              >
+                All
+              </button>
+              {this.props.organization.categories.map(category =>
+                <button
+                  key={category}
```

Comment:
Not gonna require this for this PR since it's more of a question, but is there a guarantee that the categories are unique? I'm not sure if the API validates uniqueness for tags. Could be a potential problem for use w/ keys here.

Reply:
We just ran into this using link URL for keys in the project navigation links. In a real project the key will always be unique but there is an edge case in which a project owner can accidentally create a duplicate key, in which case PFE will not render the second item. 

I think it's a sufficiently rare case that you shouldn't spend a lot of time worrying about it.

Reply:
good point. could adding `index` and then defining the key as 
```
key={`${category}-${index}`}
```
help?

Reply:
I'd strongly advise against using array index in keys, since [that breaks your component if the array order changes](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).

Reply:
sounds good